services/reference/opbnb/json-rpc-methods/subscription-methods/eth_subscribe.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: "eth_subscribe"
+todo: Is newPendingTransactions` event only mainnet supported?
+---
+
+import Tabs from "@theme/Tabs"
+import TabItem from "@theme/TabItem"
+
+import Description from "/services/reference/_partials/subscription-methods/_eth_subscribe-description.mdx";
+
+<Description />
+
+## Parameters
+
+import Params from "/services/reference/_partials/subscription-methods/_eth_subscribe-parameters.mdx";
+
+<Params />
+
+:::note
```

Comment:
Maybe add a space to be consistent with the other admonitions

Reply:
This is inconsistent across our docs (I personally prefer no spaces) but will change it here for consistency on this page.